R/compute.R

```diff
@@ -270,12 +270,9 @@ create_aks_compute <- function(workspace,
                                service_cidr = NULL,
                                dns_service_ip = NULL,
                                docker_bridge_cidr = NULL,
-                               cluster_purpose = NULL) {
+                               cluster_purpose = c("DevTest", "FastProd")) {
```

Comment:
I just confirmed w/ the inference team that cluster_purpose will default to FastProd if the argument is NULL. So if we swap the order `cluster_purpose = c("FastProd", "DevTest"))` then it will fall back to the appropriate value if user doesn't provide this argument.